draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -676,9 +676,42 @@
+
+			<t>Static values are dependent on the CPU generation, class of router and network scaling, typically the number of adjacent neighbors.
+			Examples at the time of publication are provided below. LSP Burst Size could be in the range 5 to 20. From a router perspective, this value
+			typically depends on the buffer size on the I/O path from the packet forwarding engine to the control plane which is very platform dependent.
+			It also depends upon how many IS-IS neighbors share this I/O path as typically all neighbors will send the same LSPs at the same time.
+			It may also depends on other incoming control plane traffic sharing that I/O path, how much bursty they are, and how much incoming IS-IS packets
+			are prioritized over those other incoming control plane traffic.  As indicated in <xref target="HISTORY"/>, the historical behavior from <xref target="ISO10589"/> allow a value
+			of 10 hence 10 seems conservative. From a network operation perspective, it would be beneficial for the burst size to be equal or higher than the
+			number of LSPs which may be originated by a single failure. For a node failure, this is equal to the number of IS-IS neighbors of the failed link.
+			LSP Transmission Interval could be in the range of 1 ms to 33 ms. As indicated in <xref target="HISTORY"/>, the historical behavior from <xref target="ISO10589"/> is 33ms hence
+			is conservative. The LSP Transmission Interval is an advertisement of the receiver's sustainable LSP reception rate taking into account all aspects
+			and in particular the control plane CPU and the I/O bandwidth. It's expected to improve (hence decrease) as hardware and software naturally improve
+			over time. It should be chosen relatively conservatively as this rate may be used by the sender in all conditions including worst conditions.
+			It's also not a bottleneck as the flow control algorithm may use a higher rate in good conditions, in particular when the receiver acknowledge quickly
+			and the receive window is large enough compared to the RTT.
+			LPP could be in the range of 5 to 90 with a proposed 15. A smaller value provides faster feedback at the cost of the small overhead of more PSNP messages
+			(yet same global content). The value is not susceptible to be "unsafe". PartialSNPInterval could be in the range 50ms to 500ms with a proposed 200ms.

```

Comment:
Not sure that "same global content" is necessary to say - though I understand your thought process. You are trying to point out that the number of ACKs/PSNP does not impact "correctness" - but I don't think we need to explain how the Update process works.
Also, not sure how useful "The value is not susceptible to be "unsafe"." is. This suggests (perhaps unintentionally) that the value advertised doesn't really matter. I would omit the sentence.

Reply:
1) "same global content" removed. I was trying to say that regardless of the number of PSNPs, the number of LSP-ID listed is exactly the same. (compared to TCP which may acknowledge a range of bytes)
2) I was trying to cover Mirja's comment about the risk of using too low or too high values. LPP is an important parameter to provide fast feedback to the sender but I don't think that any value may overload the receiver as it will only acks LSP when it's ready to (and it may chose to ack more or less LSP at its own will). And I don't think that it can overload the sender as at worst the sender will slow down if it has more PSNP to handle (but again the main work is probably to handle each LSP-ID inside the PSNP). I have removed that sentence as per your feedback. (as I believe that the amount of added text is already significant and hopefully good enough)